api/app/serializers/spree/api/v2/platform/menu_item_serializer.rb

```diff
@@ -5,6 +5,20 @@ module Platform
         class MenuItemSerializer < BaseSerializer
           include ::Spree::Api::V2::ResourceSerializerConcern
 
+          attributes :item_type, :code, :name, :subtitle, :destination, :new_window, :lft, :rgt, :depth
```

Comment:
@MatthewKennedy we can skip this line as `include ::Spree::Api::V2::ResourceSerializerConcern` adds those attributes automatically :)